test/test_basic_expectations.py

```diff
@@ -4,16 +4,6 @@
 
 
 class TestBasicExpectations(MockServerClientTestCase):
-    def test_expect_once_not_called_fails(self):
```

Comment:
i understand that it is tested in the verify but this serve as the documentation and i like having a clear test showing : 
if you expect 1 and you verify_expectations you will get that assertion error, same goes for the other test.

The intent is really to show the usage